src/Concordium/Client/Runner.hs

```diff
@@ -934,6 +934,33 @@ processTransactionCmd action baseCfgDir verbose backend =
                 let nrgCost _ = return $ Just $ simpleTransferEnergyCost $ Types.payloadSize encodedPayload
                 txCfg <- liftIO $ getTransactionCfg baseCfg txOpts nrgCost
 
+                let intOpts = toInteractionOpts txOpts
+                let outFile = toOutFile txOpts
+                signAndProcessTransaction_ verbose txCfg encodedPayload intOpts outFile backend
+        TransactionPLTUpdateSupply actionText amount symbolText txOpts -> do
+            baseCfg <- getBaseConfig baseCfgDir verbose
+            when verbose $ do
+                runPrinter $ printBaseConfig baseCfg
+                putStrLn ""
+
+            withClient backend $ do
+                tokenGovernanceOperation <- case actionText of
+                    "mint" -> pure $ CBOR.TokenMint amount
+                    "burn" -> pure $ CBOR.TokenBurn amount
+                    _ -> logFatal ["Only `mint` or `burn` supported for the `action` option."]
```

Comment:
Suggestion: "`Action` must be one of `mint`/`burn`."

Reply:
Changed it to:

````
`Action` option must be either `mint` or `burn`.
```